packages/edit-site/src/components/post-fields/index.js

```diff
@@ -196,23 +149,12 @@ function usePostFields() {
 				enableHiding: false,
 			},
 			{
-				label: __( 'Author' ),
-				id: 'author',
-				type: 'integer',
+				...authorField,
 				elements:
```

Comment:
The `usePostFields` hook returns an `isLoading` that is the `isLoadingAuthors`. That's why I've left the elements definition in this file.

I think this approach is ok because the final goal is to expose `usePostFields` to consumers, so they won't have to hook the data the author field anyway.

Reply:
That's a good thought, and I don't have a good answer for that right now. Another example is the `render` function of this same component: it uses the core store to get the author & image name for the given item. In that sense, the data is still "loading".

Do you think we should address this differently in this PR on is good enough to land?

Reply:
Yes, I think it can be explored separately.